packages/contact-center/task/src/IncomingTask/incoming-task.presentational.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {IncomingTaskPresentationalProps} from '../task.types';
+
+const IncomingTaskPresentational: React.FunctionComponent<IncomingTaskPresentationalProps> = (props) => {
+  const {currentTask, accept, decline, isBrowser, answered, ended, missed} = props;
+
+  if (!currentTask) {
+    return <></>; // hidden component
+  }
+
+  const callAssociationDetails = props.currentTask.data.interaction.callAssociatedDetails;
+  const {ani, dn, virtualTeamName} = callAssociationDetails;
+  const timeElapsed = ''; // TODO: Calculate time elapsed
+
+  return (
+    <div
+      data-testid="incoming-task-presentational"
+      style={{
```

Comment:
question. If we have a styles-module.scss file, why do we need styling for all these divs in the presentational file itself?